### ISSUE TITLE
Copying to clipboard doesn't work correctly in Docking branch

### ISSUE THREAD

**Issue by the-goodies** [ocornut/imgui#8254]
### Version/Branch of Dear ImGui:

Version 1.91.7 WIP, Branch: docking

### Back-ends:

example_glfw_opengl3

### Compiler, OS:

Windows 11; Visual Studio 2019

### Full config/build information:

```
Dear ImGui 1.91.7 WIP (19163)
--------------------------------
sizeof(size_t): 4, sizeof(ImDrawIdx): 2, sizeof(ImDrawVert): 20
define: __cplusplus=199711
define: _WIN32
define: _MSC_VER=1929
define: _MSVC_LANG=201402
define: IMGUI_HAS_VIEWPORT
define: IMGUI_HAS_DOCK
--------------------------------
io.BackendPlatformName: imgui_impl_glfw
io.BackendRendererName: imgui_impl_opengl3
io.ConfigFlags: 0x00000483
 NavEnableKeyboard
 NavEnableGamepad
 DockingEnable
 ViewportsEnable
io.ConfigViewportsNoDecoration
io.ConfigNavCaptureKeyboard
io.ConfigInputTextCursorBlink
io.ConfigWindowsResizeFromEdges
io.ConfigMemoryCompactTimer = 60.0
io.BackendFlags: 0x0000140E
 HasMouseCursors
 HasSetMousePos
 PlatformHasViewports
 RendererHasVtxOffset
 RendererHasViewports
--------------------------------
io.Fonts: 1 fonts, Flags: 0x00000000, TexSize: 512,128
io.DisplaySize: 1280.00,720.00
io.DisplayFramebufferScale: 1.00,1.00
--------------------------------
style.WindowPadding: 8.00,8.00
style.WindowBorderSize: 1.00
style.FramePadding: 4.00,3.00
style.FrameRounding: 0.00
style.FrameBorderSize: 0.00
style.ItemSpacing: 8.00,4.00
style.ItemInnerSpacing: 4.00,4.00
```

### Details:

Copying to clipboard doesn't work correctly in Docking branch.
When selecting all input text, it copies text correctly, but it doesn't when selecting only part of a text.
In master branch clipboard seems to work fine.

**Comment by ocornut** [comment]
My apologies, 32f1140 pushed on wednesday badly broke this (it's unrelated to branches).
Pushed fix 6982ce4

**Comment by ocornut** [comment]
Thanks for reporting!
